app/controllers/request_purpose.js

```diff
@@ -63,14 +63,6 @@ export default Ember.Controller.extend(cancelOrder, {
       }
 
       let user = this.get('user');
-      let purposeIds = [];
```

Comment:
Quick mention regarding this block that got removed. The `1` and `2` are actually hardcoded DB Ids, which means this is basically a time bomb. The IDs on my machine had changed (re-ran the seeding) and this exploded.
Luckily, this logic had already been re-written in the updated client_information page, so I just removed this.

Let's keep our eyes out and make sure that doesn't occur anywhere else ! 😄 
The appropriate practice is to load the lookup table, and generate the options based on that.

ping @Darshan-Chauhan @abhinav098 @namrataukirde @abulsayyad123 @steveyken

Reply:
Completely agree with this @patrixr . We should not add hard-coded value anywhere in code.